### PR TITLE
Config - Disable disk-benchmark in ndmv4.yaml and change batch size to 1 in default.yaml

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Disable disk-benchmark in ndmv4.yaml and change batch size to 1 in default.yaml

### CODE REVIEW

File: superbench/config/azure_ndmv4.yaml [hunk 1]
```diff
@@ -81,7 +81,7 @@ superbench:
           prefix: CUDA_VISIBLE_DEVICES={proc_rank} numactl -N $(({proc_rank}/2))
           parallel: no
     disk-benchmark:
-      enable: true
+      enable: false
```
Comment (abuccts) [inline superbench/config/azure_ndmv4.yaml:84]: update disk-benchmark in superbench/config/azure_ndv4.yaml accordingly

File: superbench/config/default.yaml [hunk 2]
```diff
@@ -164,22 +166,22 @@ superbench:
         - gpt2-large
       parameters:
         <<: *common_model_config
-        batch_size: 4
+        batch_size: 1
```
Comment (abuccts) [inline superbench/config/default.yaml:169]: pls set it in common_model_config and remove it here

File: superbench/config/default.yaml [hunk 3]
```diff
@@ -206,4 +208,4 @@ superbench:
         - vgg19
       parameters:
         <<: *common_model_config
-        batch_size: 128
+        batch_size: 1
```
Comment (abuccts) [inline superbench/config/default.yaml:211]: same

File: superbench/config/azure_ndmv4.yaml (hunk 1 above)
Comment (yukirora) [reply superbench/config/azure_ndmv4.yaml:84]: ok, updated

File: superbench/config/default.yaml (hunk 2 above)
Comment (yukirora) [reply superbench/config/default.yaml]: ok, updated

**Review by abuccts — APPROVED** [review]

**Review by cp5555 — APPROVED** [review]

**Pull request merged**